README.md

```diff
@@ -32,12 +32,12 @@ running).
 **Note:** when specifying the pod CIDR to the command below, daemonset.sh will
 generate a /24 subnet prefix to create per-node CIDRs. Ensure your pod subnet is has a
 prefix less than 24, or edit the generated ovn-setup.yaml and specify a host subnet
-prefix. For example, providing a net-cidr of "129.168.1.0/24" would require modifying
+prefix. For example, providing a net-cidr of "192.168.1.0/24" would require modifying
 ovn-setup.yaml with a host subnet prefix as follows:
 
 ```
 data:
-  net_cidr:      "192.168.1.0/24/25"
+  net_cidr:      "192.168.1.0/25"
```

Comment:
This part was right before, but the Note is confusingly written and also irrelevant now anyway. I'll submit another PR to fix that. Just remove this part for now